src/couch_mrview_http.erl

```diff
@@ -433,6 +443,7 @@ parse_params(Props, Keys) ->
 
 parse_params(Props, Keys, #mrargs{}=Args0) ->
     Args = Args0#mrargs{keys=Keys},
+    [check_compatible(K1, K2, Props) || {K1, K2} <- [{"group", "group_level"}]],
```

Comment:
@rnewson hm...that could be a problem or will make `group=false&group_level=0` a valid case.

Seems like we indeed need something `before_parse_params` that checks for such collisions. Also, there we can ensure, that every parameter was specified only once. Or we can extend `parse_params` to let it have access to params list and keep all the validation within the single place. Not sure which one is better. Tend to the first.
```